scripts/control_device.py

```diff
@@ -296,13 +296,39 @@ async def control_thermostat(
         command = remote.get_command(
             new_state, new_mode, new_target_temp, new_fan_level, new_swing, resp.state
         )
+
         printer.pprint(
             asdict(
                 await api.control_breeze_device(command),
                 verbose,
             )
         )
 
+        # Here is a special case to handle swing on special remotes
+        # if we entered here with only swing mode change, we perform it
+        if (
```

Comment:
this `if statement` seems complex, can we simplify it?

Reply:
no, it actually doesn't.
the comment may explain why is this statement there,
but it doesn't make it less complex to maintain/debug.